common/src/main/java/me/lucko/luckperms/common/locale/Message.java

```diff
@@ -3714,6 +3719,7 @@ static TextComponent prefixed(ComponentLike component) {
                                     .color(GRAY)
                                     .key("luckperms.duration.since")
                                     .args(DurationFormatter.CONCISE_LOW_ACCURACY.format(action.getDurationSince()))
+                                    .hoverEvent(HoverEvent.showText(text().append(text("Date: ", GRAY)).append(text(DATE_FORMAT.format(action.getTimestamp()), AQUA))))
```

Comment:
This `"Date:"` should be translatable and not hardcoded (see 2 lines above).
Translation terms live in `properties` files, for example the English one:
https://github.com/LuckPerms/LuckPerms/blob/master/common/src/main/resources/luckperms_en.properties

You can see some usage of the `translatable()` and `key()` methods in this class, it should be fairly simple to do!

Reply:
Good suggestion, I’ll change that soon!

Reply:
Any update? :) happy to merge this once that change has been made

Reply:
> Any update? :) happy to merge this once that change has been made

I have just commited the changes I believe. I haven't managed to check it just yet as I have no access to a server atm but I think it should be right..